File: app/src/main/java/com/example/keybindhelper/dao/StringLiveDataConverter.java
package com.example.keybindhelper.dao;

import androidx.lifecycle.MutableLiveData;
import androidx.room.TypeConverter;
import androidx.room.TypeConverters;

public class StringLiveDataConverter {
    @TypeConverter
    public static MutableLiveData<String> toLiveData(String s){
        return new MutableLiveData<>(s);
    }

    @TypeConverter
    public static String toString(MutableLiveData<String> s){
        return s.getValue();
    }
}

File: app/src/main/java/com/example/keybindhelper/dao/StringLiveDataConverter.kt
package com.example.keybindhelper.dao

import androidx.lifecycle.MutableLiveData
import androidx.room.TypeConverter

class StringLiveDataConverter {
    companion object    {
        @JvmStatic
        @TypeConverter
        fun toLiveData(s: String): MutableLiveData<String> {
            return MutableLiveData(s)
        }

        @JvmStatic
        @TypeConverter
        fun toString(s: MutableLiveData<String?>): String? {
            return s.value
        }
    }
}